grakn-kb/src/main/java/ai/grakn/factory/EmbeddedGraknSession.java

```diff
@@ -129,27 +137,19 @@ public CommitLogHandler commitLogHandler(){
      * A user should not call this method directly.
      * See {@link Grakn#session(String, String)} for creating a {@link GraknSession} for the remote API
      */
-    @SuppressWarnings("unused")//This must remain public because it is accessed via reflection
-    public static EmbeddedGraknSession create(Keyspace keyspace, SimpleURI engineUri){
-        return create(keyspace, engineUri.toString());
-    }
```

Comment:
We should keep this method, since it takes a `SimpleURI`, not a `String`

Reply:
It's unused for now, I think we should have just one method and  in the future this method should accept `SimpleURI` - but for that we should do a full refactor (given that `EmbeddedGraknSession` uses `uri` as a String )- for now having two methods was just confusing to me 🤔 🥇